src/main/java/models/SpanishGame.java

```diff
@@ -0,0 +1,31 @@
+package models;
+
+import java.util.ArrayList;
+
+/**
+ * Created by harderg on 2/14/16.
+ */
+public class SpanishGame extends Game {
+
+    /*public void buildDeck() {
+        for(int i = 1; i < 14; i++){
+            if (i < 8 || i > 10) {
+                deck.add(new Card(i, SpanishSuit.Clubs));
+                deck.add(new Card(i, SpanishSuit.Coins));
+                deck.add(new Card(i, SpanishSuit.Swords));
+                deck.add(new Card(i, SpanishSuit.Cups));
+            }
+        }
+    }*/
+
+    public void buildDeck() {
```

Comment:
Was the build deck the only thing that differentiated between the Spanish Deck and English Deck? I don't recall correctly but I feel like there was one or two more differences between the two. Besides that, I like the idea of the if statement checking for i to be the correct values.